data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/configuration/ThresholdOptions.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink.configuration;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import jakarta.validation.constraints.NotNull;
+
+/**
+    An implementation class of s3 index configuration Options
+ */
+public class ThresholdOptions {
+    static final int DEFAULT_EVENT_COUNT = 200;
+    private static final long DEFAULT_BYTE_CAPACITY = 2500;

```

Comment:
This value seems rather low.

Reply:
Thanks, David for your review. Modified with higher values.